src/main/resources/eu/europa/ted/eforms/sdk/analysis/drools/xmlNoticeRules.drl

```diff
@@ -0,0 +1,12 @@
+package eu.europa.ted.eforms.sdk.analysis.drools;
+
+unit SdkUnit;
+
+rule "Every notice example has correct SDK version"
+when
+  /sdkProject[ $expectedCustomizationId : customizationId ]
+  and
+	/xmlNotices[ $n: this, customizationId != '' && customizationId != $expectedCustomizationId ]
```

Comment:
The check "customizationId != ''" is redundant.

Also, if the suggestion regarding the SDK version being exposed as a variable (instead of as fact) is implemented, then the "when" block can be simplified as (assuming a variable "sdkVersion"):
/xmlNotices[ $n: this, customizationId != sdkVersion ]

Reply:
The check "customizationId != ''" is kept, as a notice example without any customization ID indicated should not cause an error.
The when block has been simplified after the changes for SdkMetadata.